src/descriptor/checksum.rs

```diff
@@ -43,15 +41,17 @@ fn poly_mod(mut c: u64, val: u64) -> u64 {
     c
 }
 
-/// Compute the checksum of a descriptor
-pub fn get_checksum(desc: &str) -> Result<String, DescriptorError> {
```

Comment:
I don't know if anyone is using `get_checksum()` but since it's public our policy is to deprecate it for at least one release before completely removing it. We haven't been super strict about it, but we do need to be more consistent. I think you could make a simple function to call your new `get_checksum_bytes` and convert to at String.

Reply:
@notmandatory I haven't removed it at all! And yes, `get_checksum` calls `get_checksum_bytes` 😅

Reply:
Check line 79